access-control.properties

```diff
@@ -0,0 +1,2 @@
+access-control.name=tech.stackable.trino.opa.OpaAuthorizer
+opa.uri=http://localhost:8181/
```

Comment:
We probably have to make the port (and even hostname) configurable via cli in case there is no local opa installation or its running on a different port.